tests/fuzz/fuzz_h2_decoder_correct.c

```diff
@@ -211,8 +271,24 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
             struct aws_h2_frame_priority_settings priority = s_generate_priority(&input);
             struct aws_h2_frame_priority_settings *priority_ptr = use_priority ? &priority : NULL;
 
+            /* Server can only receive request-style HEADERS, client can only receive respone-style HEADERS.
```

Comment:
response

Reply:
man I do not know why my spell checker things "respone" is a valid word